src/modules/twap/utils/deadlinePartsDisplay.ts

```diff
@@ -0,0 +1,34 @@
+import { TwapOrdersDeadline } from '../state/twapOrdersSettingsAtom'
+import ms from 'ms'
+
+const [oneD, oneH, oneM, oneS] = [ms('1d'), ms('1h'), ms('1m'), ms('1s')]
+
+function customDeadlineToTimestamp(customDeadline: TwapOrdersDeadline['customDeadline']): number {
+  const hoursToMinutes = customDeadline.hours * 60
+  const minutesToSeconds = (hoursToMinutes + customDeadline.minutes) * 60
```

Comment:
Can you use `ms()` here like you do below?

Something like:

```
const hoursToMinutes = ms(customDeadline.hours + 'h')
```

(Have not tested it!)